app/views/docs/production.phtml

```diff
@@ -27,7 +27,11 @@
 
 <p>There are three Appwrite containers that do keep their state. MariaDB, Redis, and InfluxDB containers are used for storing data, cache and pub/sub messaging, and usage stats (in this order). To scale them out, all you need to do is set up a standard cluster (same as you would with any other app using these technologies) according to your needs and performance.</p>
 
-<h2><a href="/docs/production#emails" id="emails">Emails</a></h2>
+<h2><a href="/docs/production#disable-limits" id="disable-limits">Enable Rate Limits</a></h2>
+<p>If you disabled rate limits during development, make sure you re-enable them when moving to production environments. Rate limiting can be enabled by setting the <span class="tag">_APP_OPTIONS_ABUSE</span> environment variable to <code>enabled</code>.</p>
+<p>
+<a href="/docs/environment-variables"><i class="icon-angle-circled-right"></i>Learn more about environment variables</a>
+</p>
 
 <p>Sending emails is hard. There are a lot of SPAM rules and configurations to master in order to set a functional SMTP server. The SMTP server that comes packaged with Appwrite is great for development but needs some work done to function well against SPAM filters.</p>
```

Comment:
You've missed out the <a> tag to denote the beginning of the emails section

Reply:
You're right I need to update what's written here anyway.